index.html

```diff
@@ -48,5 +48,116 @@ <h2>I am a Software Developer</h2>
       </div>
     </section>
 
+    <section class="work-section">
+      <div class="title">
+  <h1>My Recent Works</h1>
+      </div>
+    </section>
+
+ <section class="works">
+   <div class="my-works">
+     <div class="work-pic-1">
+       <img src="./img/work-img/work-1.png" alt="My work" />
+     </div>
+     <div class="work-content">
+       <h2>Multi-Post Stories</h2>
+       <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+       <div class="programming-languages">
+         <p>html</p>
+         <p>Bootstrap</p>
+         <p>Ruby</p>
+       </div>
+       <a href="#">See Project</a>
+     </div>
+   </div>
+   <div class="work-pic-2">
+     <h2>Profesional Art<br> Printing Data</h2>
+     <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+     <div class="programming-languages">
+       <p>html</p>
+       <p>Bootstrap</p>
+       <p>Ruby</p>
+     </div>
+     <a href="#">See Project</a>
+   </div>

```

Comment:
- [ ] As requested by the previous reviewer, it is required to have your project as identical to the Figma template as possible. The section should use the font and colours (background and font) defined for the template. Kindly consider adding a bit more CSS for us to achieve this and make your project rock 😃

<html>
<body>
<!--StartFragment--><p dir="auto" style="box-sizing: border-box; margin-top: 0px !important; margin-bottom: 16px; color: rgb(201, 209, 217); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-size: 14px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(13, 17, 23); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"></p>

Figma design | Your design
-- | --
![image](https://user-images.githubusercontent.com/29071984/170837269-6c0dc6a2-1937-4ff9-aea0-75e3120232fb.png) | ![image](https://user-images.githubusercontent.com/29071984/170837322-3ac450ba-ce76-4e8e-b41b-7ec98d39c8fb.png)
 -- | ![image](https://user-images.githubusercontent.com/29071984/170837374-a3d9b65e-d558-47cb-8dd1-33008beb5c17.png)




<!--EndFragment-->
</body>
</html>